gen/testvars/fvEpIpTag.yaml

```diff
@@ -46,3 +46,5 @@ parents:
     parent_dependency: ""
     parent_dn: "aci_tenant.test.id"
     class_in_parent: false
+
+targets:
```

Comment:
Do we need this when there are no targets? We can probably remove, same counts for others files.